src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,35 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    font-size: 12px;
+    line-height: 14px;
+    font-weight: 500;
+    text-transform: uppercase;
```

Comment:
Make sure to use the font sizes and line heights from the variables file, to maintain consistency and ease of maintenance. If these values are not in the variables file yet, consider adding them.